contents/blog/hotjar-for-mobile-apps-alternatives.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: 'Hotjar alternatives for mobile apps (iOS & Android)'
+date: 2024-12-10
+author:
+  - lior-neu-ner
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/clarity-alternatives/clarity-alternatives.png
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+While [Hotjar](/blog/posthog-vs-hotjar) is an excellent tool for viewing session replays and understanding user behavior, it doesn't support mobile apps. 
+
+In this post, we compare the best alternatives for mobile so that you can find the best replay tool for your needs.
```

Comment:
```suggestion
Hotjar is one of the most popular session replay tools on the market, but it doesn't work on iOS and Android mobile apps, or support popular app frameworks like React Native and Flutter. 

This guide compares replay tools that support mobile apps, so you can find the best replay tool for your needs. If you don't need a "Hotjar for mobile", see our guide to [Hotjar alternatives for web apps](https://posthog.com/blog/best-hotjar-alternatives).
```

Reply:
We should add a link to this guide from the Hotjar alternatives guide with the anchor text "Hotjar for mobile". Linking between them like this should help Google understand which page to rank for our desired term and prevent internal conflict.